<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RESDPcgrader</class>
 <widget class="QDialog" name="RESDPcgrader">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1391</width>
    <height>850</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="labelMemTransferText">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Memory Transfer results for a 500MB file:</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelMemtrsRuns">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>110</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelMemtrsTimeExec">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>140</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelMemtrsMBRate">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>170</y>
     <width>201</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelMemaccText">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>80</y>
     <width>381</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Results for Memory Access performed on a 4096x4096 square matrix:</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelMemaccRuns">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>110</y>
     <width>211</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelMemaccTimeBest">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>140</y>
     <width>221</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelMemaccOpsBest">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>170</y>
     <width>241</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelMemaccTimeWorst">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>200</y>
     <width>231</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelMemaccOpsWorst">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>230</y>
     <width>201</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelIntText">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>80</y>
     <width>321</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Results for Encryption and Decryption on a 64MB file:</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelIntRuns">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>110</y>
     <width>231</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelIntTimeExec">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>140</y>
     <width>241</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelIntOpsExec">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>170</y>
     <width>221</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelIntTimeWrite">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>200</y>
     <width>221</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelFlopText">
   <property name="geometry">
    <rect>
     <x>1020</x>
     <y>80</y>
     <width>321</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Results of computing the Mandelbrot set for 1000 iterations:</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelFlopRuns">
   <property name="geometry">
    <rect>
     <x>1020</x>
     <y>110</y>
     <width>241</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelFlopTimeExec">
   <property name="geometry">
    <rect>
     <x>1020</x>
     <y>140</y>
     <width>271</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelFlopOpsExec">
   <property name="geometry">
    <rect>
     <x>1020</x>
     <y>170</y>
     <width>261</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>380</y>
     <width>731</width>
     <height>461</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>Tab 1</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="tab_4">
    <attribute name="title">
     <string>Tab 2</string>
    </attribute>
   </widget>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>1361</width>
     <height>211</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Shape::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Shadow::Raised</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSaveCharts">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>360</y>
     <width>121</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Save Charts</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>310</y>
     <width>81</width>
     <height>51</height>
    </rect>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarPolicy::ScrollBarAlwaysOff</enum>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe UI'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:26pt;&quot;&gt;10.0&lt;/span&gt;&lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:26pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>310</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Final Score:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <zorder>frame</zorder>
  <zorder>tabWidget</zorder>
  <zorder>labelMemaccTimeBest</zorder>
  <zorder>labelMemaccTimeWorst</zorder>
  <zorder>labelIntOpsExec</zorder>
  <zorder>labelMemtrsMBRate</zorder>
  <zorder>labelIntText</zorder>
  <zorder>labelMemtrsRuns</zorder>
  <zorder>labelIntTimeExec</zorder>
  <zorder>labelMemaccText</zorder>
  <zorder>labelMemaccOpsWorst</zorder>
  <zorder>labelFlopRuns</zorder>
  <zorder>labelIntTimeWrite</zorder>
  <zorder>labelMemtrsTimeExec</zorder>
  <zorder>labelMemaccRuns</zorder>
  <zorder>labelMemaccOpsBest</zorder>
  <zorder>labelMemTransferText</zorder>
  <zorder>labelFlopText</zorder>
  <zorder>labelFlopOpsExec</zorder>
  <zorder>labelIntRuns</zorder>
  <zorder>labelFlopTimeExec</zorder>
  <zorder>btnSaveCharts</zorder>
  <zorder>textEdit</zorder>
  <zorder>label</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
